internal/providers/terraform/aws/codebuild_project.go

```diff
@@ -0,0 +1,98 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetCodebuildProjectRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_codebuild_project",
+		RFunc: NewCodebuildProject,
+	}
+}
+
+func NewCodebuildProject(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := d.Get("region").String()
+
+	environmentComputeType := d.Get("environment.0.compute_type").String()
+	environmentType := d.Get("environment.0.type").String()
+
+	var monthlyBuildMinutes int64
+	if u != nil && u.Get("monthly_build_minutes").Exists() {
+		monthlyBuildMinutes = u.Get("monthly_build_minutes").Int()
+	}
+
+	if environmentComputeType == "BUILD_GENERAL1_SMALL" {
+		if monthlyBuildMinutes <= 100 {
+			return &schema.Resource{
+				NoPrice:   true,
+				IsSkipped: true,
+			}
+		}
+		monthlyBuildMinutes -= 100
+	}
```

Comment:
We currently ignore the free tier for each resource since we don't know if it is being used up by resources in other Terraform projects. So for just now I would remove this logic. In the future we might add an option to include the free tier or not.